blessed/sequences.py

```diff
@@ -15,7 +16,10 @@
 import wcwidth
 import six
 
-__all__ = ('init_sequence_patterns', 'Sequence', 'SequenceTextWrapper',)
+__all__ = ('init_sequence_patterns', 'Sequence', 'SequenceTextWrapper', 'iter_parse')
+
+
+TextFragment = collections.namedtuple('TextFragment', ['ucs', 'is_sequence', 'capname', 'params'])
```

Comment:
Changed my mind about ``capname``, now wish for it to be ``name``, given the vanilla SGR matches as well as the wish to give phrase names such as 'newline' for ``\n`` -- we're not really always providing a capability name any longer.